api/controllers/FileUploadController.js

```diff
@@ -0,0 +1,133 @@
+const multer = require('multer');
+const sails = require('sails');
+
+const uploadFileToStorage = require('../helpers/uploadFileToStorage');
+const deleteFileFromStorage = require('../helpers/deleteFileFromStorage');
+const {
+    virusScanFile,
+    checkTypeSizeAndDuplication,
+    displayErrorAndRemoveLargeFiles,
+    connectToClamAV,
+} = require('../helpers/uploadedFileErrorChecks');
+
+const MAX_FILES = 50;
```

Comment:
Again, maybe make this an env variable. We occasionally get requests to limit applications if the office are very busy so any way to make this kind of thing configurable is good.